projects/atomic-swap-service/headless-wallet.md

```diff
@@ -71,20 +85,23 @@ This will allow the wallet user to add any existing proposal to the wallet's `li
 
 To ensure the informed parameters are correct, a `get` request will also be made at this time to the service backend, and a validation executed on the received contents. Any failure found on the process will be thrown and the registration discarded.
 
-### Fetch
+### Listened Proposals endpoint
+A new route `[GET] /wallet/atomic-swap/tx-proposal/listenedProposals` will be created, allowing the user to retrieve the list of `proposalIds` currently being listened to. No password should be retrieved on this call.
```

Comment:
I would call it `registered-proposals`, because you used `register` in the other API

Reply:
Changed it to `/list` , based on a suggestion from a comment above.